mlir/tools/mlir-tblgen/EnumPythonBindingGen.cpp

```diff
@@ -165,16 +164,27 @@ static bool emitDialectEnumAttributeBuilder(const AttrOrTypeDef &attr,
 static bool emitPythonEnums(const llvm::RecordKeeper &recordKeeper,
                             raw_ostream &os) {
   os << fileHeader;
+  for (const auto &it :
+       recordKeeper.getAllDerivedDefinitionsIfDefined("EnumAttr")) {
+    EnumAttr *enumAttr;
+    for (const auto &value : it->getValues())
+      if (value.getType()->getAsString() == "EnumAttrInfo")
```

Comment:
there must be a better way to do this but for the life of me i can't figure it out.